src/components/ProtectedRoute.tsx

```diff
@@ -1,23 +1,14 @@
 import React from 'react';
 import { Navigate } from 'react-router-dom';
 import { useAuth } from '../hooks/useAuth';
-import { Center, Spinner } from '@chakra-ui/react';
 
 // Define the props for ProtectedRoute
 interface ProtectedRouteProps {
   children: React.ReactNode;
 }
 
 const ProtectedRoute: React.FC<ProtectedRouteProps> = ({ children }) => {
-  const { isAuthenticated, member } = useAuth();
-
-  if (member === undefined) {
-    return (
-      <Center>
-        <Spinner />
-      </Center>
-    );
-  }
+  const { isAuthenticated } = useAuth();
```

Comment:
This doesn't forward people to a different page because of isAuthenticated starting as false?

Reply:
Nope, as you can see it goes immediately from page to page after initial login redirect

<img width="558" alt="Screenshot 2024-08-18 at 2 30 37 PM" src="https://github.com/user-attachments/assets/f48a3556-2877-4729-ae52-c085c58b19b4">
<img width="545" alt="Screenshot 2024-08-18 at 2 31 17 PM" src="https://github.com/user-attachments/assets/5fc4e72a-5ada-438c-96cb-8f959aa70666">
<img width="1083" alt="Screenshot 2024-08-18 at 2 31 49 PM" src="https://github.com/user-attachments/assets/57b60929-3337-4022-a247-55f507a06b47">
<img width="1086" alt="Screenshot 2024-08-18 at 2 32 01 PM" src="https://github.com/user-attachments/assets/35beff62-0e27-48eb-8c74-3fbdc6f81881">
<img width="1093" alt="Screenshot 2024-08-18 at 2 32 19 PM" src="https://github.com/user-attachments/assets/3d844912-344e-47b0-b29b-fc23902d735c">
<img width="1081" alt="Screenshot 2024-08-18 at 2 32 33 PM" src="https://github.com/user-attachments/assets/5727783f-6823-4256-b28b-ce0cf4e32b4e">
<img width="1089" alt="Screenshot 2024-08-18 at 2 32 46 PM" src="https://github.com/user-attachments/assets/97a50449-5aeb-4325-8e61-eae98e312291">
<img width="1080" alt="Screenshot 2024-08-18 at 2 32 58 PM" src="https://github.com/user-attachments/assets/59b6487f-44a4-4774-a999-7983e87bfbfa">

and so on